components/places/src/storage/history.rs

```diff
@@ -405,14 +413,18 @@ pub fn prune_destructively(db: &PlacesDb) -> Result<()> {
 }
 
 pub fn prune_older_visits(db: &PlacesDb) -> Result<()> {
-    let _tx = db.begin_transaction()?;
```

Comment:
oops - this was a bug that the tx wasn't committed, right?

Reply:
I believe it gets committed when `_tx` gets dropped.

Reply:
https://docs.rs/rusqlite/0.12.0/rusqlite/struct.Transaction.html#method.drop_behavior says "Transactions will roll back by default. Use commit method to explicitly commit the transaction, or use set_drop_behavior to change what happens when the transaction is dropped." and I can't see where we call `set_drop_behavior`.